src/test/java/org/opensearch/action/admin/indices/mapping/get/IndexForecasterActionHandlerTests.java

```diff
@@ -1427,182 +1409,4 @@ public void testCreateMappingException() throws InterruptedException, IOExceptio
         }));
         assertTrue(inProgressLatch.await(10, TimeUnit.SECONDS));
     }
-
```

Comment:
you removed this test, are we checking this somewhere else?

Reply:
this branch is no longer being tested here since AD no longer has knowledge of feature flag OPENSEARCH_RESOURCE_SHARING_ENABLED. It was added here specifically to cover that scenario. 

if you look index IndexForecasterActionHandler.java, you will see that I've remove relevant code as well. We no longer automatically share the resource with backend_roles upon creation, with the new authz scheme. Users will be required to explicitly share the resource. 
A share API is en-route to be added to Security plugin.

AD also has an option to add its own /share api however security is the ideal place as it avoids multiple implementations by different plugins to achieve same result.